src/routes/Auth/Auth.ts

```diff
@@ -0,0 +1,26 @@
+import StatusCodes from 'http-status-codes';
+import { Request, Response, Router } from 'express';
+//import { paramMissingError, IRequest } from '@shared/constants';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import userModel from '../../entities/User/User.schema';
+import validateAuthReq from '../../entities/Auth/Auth.validation';
+
+const router = Router();
+const { BAD_REQUEST } = StatusCodes;
+const User = userModel;
+
+// authenticate users
+// eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
this is not a good way to handle eslint warnings. 
It should be used as a hint to find where something is not right. 
I am 95% sure you can hover over the warnings and see some valuable feedback from linter, why the code gives warnings